frontend/src/Tracks.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Button from './components/Button';
+import Title from './components/PageTitleHeader';
+import TrackTile from './components/TracksTile';
+import { ReactComponent as TracksIcon } from './assets/icons/record-thicc.svg';
+import Artistheader from './components/ArtistHeader';
+
+export default function Tracks() {
+  return (
+    <div>
```

Comment:
Looks like you're returning a div around the whole component. You can actually return `<Fragment>` with everything inside to avoid the need for an additional `<div>` tag `import React, { Fragment } from 'react';`